app/models/user.rb

```diff
@@ -101,7 +101,7 @@ class User < ApplicationRecord
   scope :system, -> { where(roles: {name: "System"}).joins(:active_roles) }
 
   scope :staff, -> { where(roles: {name: ["Supervisor", "Reviewer"]}).joins(:active_roles) }
-  scope :except_stockit_user, -> { where.not(first_name: "Stockit", last_name: "User") }
```

Comment:
is that old version no longer working ?

Reply:
yeah.
multiple `not` conditions needs to be separated henceforth
for example `where.not(first_name: 'Anakin', last_name: 'Skywalker')` should be 
`where.not(first_name: 'Anakin').where.not(last_name: 'Skywalker')`